CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added connection/disconnection simulation to the `yarpctest` fake transport
   and peers.
 - x/yarpctest: Added support for specifying outbound middleware.
+- x/yarpctest: Changed `fakePeer` id to use interface.
```

Comment:
we actually have two `yarpctest` packages, in in `x/` and one at the top level. You want the top level one (not the `x/`)

Reply:
thanks!